examples/scheduling.rst

```diff
@@ -86,7 +86,7 @@ find solutions that meet all the constraints.
    ====================  =================  ==============  ==================  =================
 
 Ocean's :std:doc:`dwavebinarycsp <binarycsp:index>` enables the
-definition of constraints in different way, including by defining functions that evaluate
+definition of constraints by defining functions that evaluate to 
```

Comment:
I think the cause of any lack of clarity you found here is that there's a typo: "way" should be "ways". Your rewrite is accurate but it might be a bit misleading in suggesting that there is just one way of defining constraints. How about, "Ocean’s dwavebinarycsp enables the definition of constraints in different ways. You can define functions that evaluate True when the constraint is met."

Reply:
Got it.